### ISSUE TITLE
Why did he deleted it?

### ISSUE THREAD

**Issue by YINDSTR17** [Juaasey/afsscript#1]
anyone has an idea if he's going to reupload the script and why did he removed it?

**Comment by Juaasey** [comment]
I have no idea, I was tryna find shrawu script with the new version but It was too late. He deleted every single script, including discord deleted server that he's owners. Im also have no idea, Whom I'm tryna find auto frm trial, every script afs is sucks.

**Comment by YINDSTR17** [comment]
i found his discord, Shrawu#0001 dms are closed sadly, also found email, im trying to contact him, if u want it too here it is. mem@mem.rip

**Comment by YINDSTR17** [comment]
i think his github got deleted lol i cant find username

**Comment by Juaasey** [comment]
i dont think that his email, and yes his/her github deleted. i'll be sad thatI like his script is way more useful than others especially no one ever make auto farm trial.

**Comment by YINDSTR17** [comment]
i found it in the .txt bro

**Comment by YINDSTR17** [comment]
of the script

**Comment by YINDSTR17** [comment]
also i found his roblox account, lolnoobtwt he is verified in blox fruits server

**Comment by Juaasey** [comment]
if it same version of txt, its noot gonna work it'd sorta connect to github stuff.

**Comment by YINDSTR17** [comment]
its other version

**Comment by YINDSTR17** [comment]
it has auto farm and auto trial but autoskip not working and no auto raid too

**Comment by Juaasey** [comment]
really? Send me Kuroovop#0001. or in here. would be so nice.

**Comment by YINDSTR17** [comment]
sure.
